CMakeLists.txt

```diff
@@ -54,6 +54,10 @@ else()
   message("Doing a local build")
 endif()
 
+if($ENV{NUPIC_ARM_BUILD})
```

Comment:
Yes, that is what we do for all other platforms so please do it here as well. What is the value of `CMAKE_SYSTEM_NAME` on ARM?